airflow/providers/amazon/aws/utils/emailer.py

```diff
@@ -35,9 +37,22 @@ def send_email(
     **kwargs,
 ) -> None:
     """Email backend for SES."""
+    from_email = kwargs.get('from_email') or os.environ.get('SES_MAIL_FROM')
+    if not from_email and conf.has_option('ses', 'ses_mail_from'):
+        from_email = conf.get('ses', 'ses_mail_from')
+
+    from_name = kwargs.get('from_name') or os.environ.get('SES_MAIL_SENDER')
+    if not from_name and conf.has_option('ses', 'ses_mail_sender'):
+        from_name = conf.get('ses', 'ses_mail_sender')
+
+    if from_email:
+        mail_from = formataddr((from_name, from_email))
+    else:
+        mail_from = None
+
     hook = SESHook(aws_conn_id=conn_id)
     hook.send_email(
-        mail_from=None,
+        mail_from=mail_from,
```

Comment:
Wouldn’t think have the exact same error if none of the above were set? I feel we should raise an exception right here if `mail_from` is None so it’s moer obvious to debug.